src/index.js

```diff
@@ -701,7 +711,10 @@ const members = {
     get () {
       let textContent = '';
       eachChildNode(this, function (node) {
-        textContent += node.textContent;
+        const isCommentNode = node.nodeType === 8;
```

Comment:
Instead of creating a separate variable, you can probably get the same point across using the `Node.COMMENT_NODE` constant.